src/app/not-found.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Link from 'next/link';
+import NotFound from '@icons/notFound.png';
+import Image from 'next/image';
+
+const NotFoundPage = () => (
```

Comment:
🔴Please follow the export and component declaration conventions established.
Use the following way of exporting
```js
export default function NotFoundPage() {
   // ....
}
```